util/logtools/make_timeline.ts

```diff
@@ -541,41 +553,52 @@ const assembleHeaderZoneInfoStrings = (fight: FightEncInfo): string[] => {
     headerInfo.push(zoneNameLine);
   }
   if (zoneId !== undefined) {
-    const zoneIdLine = `# ZoneID: ${zoneId}`;
+    const zoneIdLine = `# ZoneId: ${zoneId}`;
     headerInfo.push(zoneIdLine);
   }
   if (headerInfo.length > 0)
     headerInfo.push('');
   return headerInfo;
 };
 
-const assembleHeaderAbilityStrings = (
+const assembledIgnoreHeaderStrings = (
   args: ExtendedArgs,
-  encounterAbilityList: { [string: string]: string },
+  encounterAbilityList: EncounterAbilityList,
 ): string[] => {
   const assembled = [];
-  const sortedIgnore = args.ignore_id?.sort();
-  if (sortedIgnore !== undefined) {
-    const iiLine = `# -ii ${sortedIgnore.join(' ')}\n`;
+
+  const sortedCombatantIgnore = args.ignore_combatant?.sort();
+  if (sortedCombatantIgnore !== undefined) {
+    const joinedIgnore = sortedCombatantIgnore.map((x) => `"${x}"`).join(' ');
+    assembled.push(`# -ic ${joinedIgnore}`);
+  }
+
+  const sortedAbilityIgnore = args.ignore_id?.sort();
+  if (sortedAbilityIgnore !== undefined) {
+    const iiLine = `# -ii ${sortedAbilityIgnore.join(' ')}\n`;
     assembled.push(iiLine);
     assembled.push('# Ignored Abilities');
-    for (const id of sortedIgnore) {
+    for (const id of sortedAbilityIgnore) {
```

Comment:
Sorry for this nitpicky opinion, but I kind of feel like the `-ii` comment should go at the top, but the `# Ignored Abilities` should go at the bottom with the rest of the table.

Reply:
Same opinion for the `-ic` comment.  I would like to be able to recreate the entire command line string from one location, preferably in the header.  I think only the ability table(s) should be at the bottom.

Reply:
Sure thing, we can do that.